kritis-charts/templates/kritis-server-deployment.yaml

```diff
@@ -23,8 +23,8 @@ spec:
       - name: {{ .Values.image.name }}
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
-        args: ["--tls-cert-file=/var/tls/cert",
-               "--tls-key-file=/var/tls/key",
+        args: ["--tls-cert-file=/var/tls/tls.crt",
```

Comment:
The secret will have data fields "cert" and "key" since you are using kubectl create tls --cert --key command.